internal/oci/runtime_low_level.go

```diff
@@ -31,8 +31,6 @@ func NewLowLevelRuntime(logger logger.Interface, candidates []string) (Runtime,
 	if err != nil {
 		return nil, fmt.Errorf("error locating runtime: %v", err)
 	}
-
-	logger.Infof("Using low-level runtime %v", runtimePath)
```

Comment:
maybe just move it to TraceF?

Reply:
I wanted to rather control when I log this at a higher level instead of doing this for all calls to this contructor.